### DISCUSSION TITLE
"←[33mWARNING←[0m:  Invalid HTTP request received." appearing in console

### DISCUSSION THREAD

**Discussion by chook100** [zauberzeug/nicegui#966] [Q&A] [answered]
### Question

Hello.
For the past 12 hours while my NiceGUI script was running, I was getting this warning in my console:
`←[33mWARNING←[0m:  Invalid HTTP request received.`

I have received this ~6 times within the past ~12 hours. It does not appear to be affecting my script in a negative way, but nonetheless, I would like to know what exactly this means, or if something else is going on that needs my attention.
Any input would be great.
Thanks!

**Comment by rodja** [comment, marked as answer]
Are you running a public server, eg. is your NiceGUI App reachable over the internet? There are many strange requests happening. Maybe you can increase the uvicorn loglevel to see more details about these requests...

**Reply by chook100** [reply]
Yes, it is. Pretty sure now it's a port scan attack, as my router firewall did catch some dangerous IP addresses the same night. I've made the site private for now, but I'll check out the uvicorn loglevel, and also see if there is a quick-fix while my site is still in preproduction. At release, I will probably use something like cloudflare to eliminate this. Thanks for your assistance.
